cernopendata/modules/theme/webpack.py

```diff
@@ -27,21 +27,22 @@
 from invenio_assets.webpack import WebpackThemeBundle
 
 
-js = WebpackThemeBundle(
+theme = WebpackThemeBundle(
     __name__,
     'assets',
-    default='bootstrap3',
+    default='semantic-ui',
     themes={
-        'bootstrap3': dict(
+        'semantic-ui': dict(
             entry={
-                'cernopendata_js': './js/app.js',
+                'cernopendata_css': './scss/styles.scss'
             },
             dependencies={
-                'angular': '~1.4.9',
                 'jquery': '~1.9.1',
```

Comment:
Moderate: How is this working exactly? Is `jquery` included in the CSS bundle? Don't we need a JS bundle to load these deps?

PS. Isn't `jquery` being loaded by [`invenio-theme`](https://github.com/inveniosoftware/invenio-theme/blob/d94bf4a5452a31214b1485bf2225b57de93799bb/invenio_theme/webpack.py#L70)?

Reply:
Thanks for the catch, removing `jquery` as it indeed comes from `invenio-theme`.

Reply:
P.S. We will probably be breaking things because Invenio is installing `jquery` `3.2.1`, while here we require `1.9.1`. Perhaps to be investigated on a per-case basis?